docs/porting/psa/spm.md

```diff
@@ -6,8 +6,19 @@ For more information about SPM, please refer to [the SPM overview page](../apis/
 
 <span class="notes">This page gives guidelines for silicon partners adding SPM capabilities.</span>
 
+mbed-os 
+
 ### New target configuration
 
+#### Platform types
+
+- Non-PSA platform: These are single core ARMv7-M targets. On these targets, the Mbed implementation of PSA provides the same PSA services exposing PSA APIs as it would on PSA targets. The PSA emulation layer allows seamless software portability to more security-oriented targets.
+- Asymmetric Multiprocessing (AMP) systems: Multicore ARMv7-M targets (for example, PSoC6 featuring CM4 and CM0+ cores). On these targets, one of the cores is dedicated to PSA use only and implements SPE.
+- ARMv8-M: Generation of ARM processors featuring TrustZone-M architecture. PSA support for this platforms is based on *specialized* [TrustedFirmware-M](https://www.trustedfirmware.org) implementation.
+
+
+#### JSON target definition
+
```

Comment:
Should we add a link to this page: https://os.mbed.com/docs/mbed-os/development/porting/porting-targets.html?